hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -856,14 +876,16 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     else {
       this.fetchMismatchContainers(this.state.DEFAULT_LIMIT, this.state.prevKeyMismatch, this.state.mismatchMissingState);
     }
+  })
   };
 
   itemRender = (_: any, type: string, originalElement: any) => {
     if (type === 'prev') {
-      return <div>{this.state.prevClickable ? <Link to="/Om" onClick={this.fetchPreviousRecords}> Prev</Link>: <Link to="/Om" style={{ pointerEvents: 'none' }}>No Records</Link>}</div>;
+      return <div>{this.state.prevClickable && this.state.pageDisplayCount ? <Link to="/Om" onClick={this.fetchPreviousRecords}> {'<< '} 
+        </Link> : <Link to="/Om" style={{ pointerEvents: 'none', color:"rgba(0,0,0,0.3)" }}>No Records</Link>}</div>;
     }
     if (type === 'next') {
-      return <div> {this.state.nextClickable ? <Link to="/Om" onClick={this.fetchNextRecords}> {'>>'} </Link> : <Link to="/Om" style={{ pointerEvents: 'none' }}>No More Further Records</Link>}</div>;
+      return <div> {this.state.nextClickable ? <Link to="/Om" onClick={this.fetchNextRecords}> {this.state.pageDisplayCount}&nbsp;&nbsp;{'   >>'} </Link> : <Link to="/Om" style={{ pointerEvents: 'none' }}>No More Further Records</Link>}</div>;
```

Comment:
Can we change `No More Further Records` to the same `No Records` for consistency?

Reply:
@devabhishekpal 

Please see changes updated changes in recent commit.